studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -1,135 +1,147 @@
-import { get } from "lodash"
-import { Filters, LogsTableName, SQL_FILTER_TEMPLATES } from "."
+import dayjs from 'dayjs'
```

Comment:
just a heads up this will conflict with https://github.com/supabase/supabase/pull/6450/files i think?
Unless 6450 was branched out of this?

Reply:
his own PR as well 😂

Reply:
oops 😄